lib/autoload/globals/Iter.ngs

```diff
@@ -358,6 +358,21 @@ F JsonData(i:Iter) collector i.each(collect + JsonData)
 
 type SkipError(Error)
 
+doc Returns the given Iter but skipping leading elements while the pattern matches. 
+doc If all the elements matches the pattern the Iter will be empty.
+doc %EX - [0, 2, 3, 25, 19, 7].Iter().skip(F(x) x != 3).Arr() returns [3, 25, 19, 7]
+doc %RET - Iter
+F skip(i:Iter, pattern=Bool.constructors){
+	while i {
+		i.peek() !~ pattern breaks
+		i.next()
+	}
+	i
+}
+
+TEST [1, 2, 3, 4, 5, 34, 5].Iter().skip(F(x) x != 3).Arr() == [3, 4, 5, 34, 5]
+TEST [1, 2, 3,5, 34, 5].Iter().skip(F(x) x != 4).Arr() == []
```

Comment:
* `F(x) x != 3` should be `X != 3` or `Not(3)`
* Similarly with `4`
* `3,5` -> `3, 5`